### ISSUE TITLE
Blinds control

### ISSUE THREAD

**Issue by tmarquespt** [andrei-tatar/node-red-contrib-nora#9]
Hi, is there a way we can controll blinds with a command like open/close X blind or open 30% without using the light entity?

**Comment by andrei-tatar** [comment]
No. There is no support for blinds in Google Home: https://developers.google.com/actions/smarthome/guides/

I also asked for support here: https://github.com/actions-on-google/smart-home-nodejs/issues/148

**Comment by caesar907** [comment]
I'm using gBridge for my Node Red ( https://github.com/kservices/gBridge ). It somehow seems to support shutters already?

**Comment by andrei-tatar** [comment]
@caesar907 I don't see shutters in the supported devices... Is there another list?
![image](https://user-images.githubusercontent.com/6473183/51776026-4f8f0380-2100-11e9-9049-2f6e4d524cdf.png)

**Comment by caesar907** [comment]
I didn't see that. Well, let's see whether its actually going to brejk

**Comment by DenDeze** [comment]
In the home app my blinds cannot be opened or closed, only by voice

**Comment by Hypnos3** [comment]
I have the same problem, but I think that's up to Google.
I would only care if it is language dependent.

Can English-language users also not open or close the blinds in the Google Home app?

**Comment by andrei-tatar** [comment]
No, it's not up to me. Yes, it's the same for all (and not just the blinds).

**Comment by frogale** [comment]
Still not working. Only voice control is possible. Do you know more about this google bug?

**Comment by andrei-tatar** [comment]
Only some devices support touch control. You can see the list here:
https://developers.google.com/assistant/smarthome/develop/touch-controls#supported_device_types

**Comment by Barabba11** [comment]
I think i't a good idea if all of us ask Google Home support (you find feedback into the app) to introduce this function. Eventually worth to write a line them also on Facebook. Thanks
